turbinia/processors/partitions.py

```diff
@@ -40,6 +41,12 @@ def Enumerate(evidence):
   options = volume_scanner.VolumeScannerOptions()
   options.partitions = ['all']
   options.volumes = ['all']
+  if location:
+    log.debug('Parsing for partition at location {0!s}'.format(location))
+    if location.find('apfs') != -1 or location.find('lvm') != -1:
```

Comment:
Why do we need to do this?  Can we maybe add a comment about this?

Reply:
Added a comment. APFS and LVM are volumes rather than partitions.